sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -405,9 +423,10 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     mapboxRenderer.map = null
   }
 
-  companion object {
-    private const val TAG = "Mbgl-RenderThread"
-    private val ONE_SECOND_NS = 10.0.pow(9.0).toLong()
-    private val ONE_MILLISECOND_NS = 10.0.pow(6.0).toLong()
+  private companion object {
+    const val TAG = "Mbgl-RenderThread"
+    const val RETRY_DELAY_MS = 50L
```

Comment:
will it work for all devices?
in my experience, such delays don't work well on different devices, especially older ones

Reply:
we don't care about actual time limit. in any case current logic is to re-schedule infinitely. delay is needed to free render message queue a bit

Reply:
Can you explain why you choose **50ms** please? 
IIRC one frame is around 33ms, should have 1-2 frames delay?

Reply:
@axti I guess I did answer it in https://github.com/mapbox/mapbox-maps-android/pull/1068#discussion_r788845168 😄 In general 50 ms is just a number out of my head. Sticking to 33 ms makes no sense as:
1. 90 / 120 Hz screens are present today meaning 33 ms is not a strict constant.
2. Frame time is not connected with actual Android SDK timings, we give that delay for system try to recover after some driver errors (e.g. could not obtain EGL surface, set EGL context) or e.g. waiting for Android to provide us new `Surface` object.

Reply:
@kiryldz please add this explanation as a comment to avoid confusion in future